packages/manager/apps/pci-gateway/src/pages/add/NetworkStep.tsx

```diff
@@ -168,11 +172,23 @@ export const NetworkStep = (): JSX.Element => {
       });
     },
     onError: (error) => {
-      addError(
-        t('pci_projects_project_public_gateways_add_error', {
-          message: error,
-        }),
-      );
+      if (error.class.includes(TGatewayErrorClass.MaxQuotaReached)) {
```

Comment:
`error` is any here.

As I wrote in LBAAS pull request, you should handle error object with a type guard as `AxiosError` does not guarantee you to expose a `class` property in generic `data` object